packages/paste-website/src/pages/components/label/index.mdx

```diff
@@ -0,0 +1,354 @@
+---
+title: Label - Components
+package: '@twilio-paste/label'
+description: Label used to name Paste form fields.
```

Comment:
My only feedback is that we should come up with a better package description for here and in the `package.json` file. 

Let's get the team to have input on it and get consensus first, before we push up an update though. We can do it in the `plz` slack channel on this review item

Reply:
Sounds good. Thanks for kicking that off.

Reply:
Updated.